### PR TITLE
docs: describe validator scheduling

### PR DESCRIPTION
**Pull request by pro-wh**
from https://github.com/oasisprotocol/oasis-core/pull/3068#pullrequestreview-440226214

> a description for how the validator committee is selected

### CODE REVIEW

File: docs/consensus/scheduler.md [hunk 1]
```diff
@@ -19,6 +19,19 @@ the [consensus service API documentation].
 
 ## Validator Committee
 
+To schedule the validator committee, the committee scheduler selects among
+nodes registered with the `RoleValidator` role.
+Each node's entities must have an [escrow account balance] meeting the total
```
Comment (peterjgilbert) [inline docs/consensus/scheduler.md:24]:
```suggestion
Each node's entity must have an [escrow account balance] meeting the total
```

File: docs/consensus/scheduler.md [hunk 2]
```diff
@@ -19,6 +19,19 @@ the [consensus service API documentation].
 
 ## Validator Committee
 
+To schedule the validator committee, the committee scheduler selects among
+nodes registered with the `RoleValidator` role.
```
Comment (kostko) [inline docs/consensus/scheduler.md:23]: Link `registered` to `registry.md#register-node`. And link `RoleValidator` to https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/common/node?tab=doc#RoleValidator.

File: docs/consensus/scheduler.md [hunk 3]
```diff
@@ -19,6 +19,19 @@ the [consensus service API documentation].
...
+Each node's entities must have an [escrow account balance] meeting the total
+thresholds for the nodes and runtimes that it has registered.
+If an entity's escrow account balance is too low to meet the total threshold,
+the committee scheduler does not consider that entity's nodes.
+
+From these qualifying nodes, the committee scheduler selects at most one node
+from each entity, up to a maximum validator committee size.
+Unlike how the committee scheduler schedules other committees, it schedules the
+validator committee by choosing nodes from the entities that have the highest
```
Comment (kostko) [inline docs/consensus/scheduler.md:32]: Maybe mention (and link to) parameters which specify K.

File: docs/consensus/scheduler.md (hunk 2 above)
Comment (pro-wh) [reply docs/consensus/scheduler.md:23]: oh good suggestions

File: docs/consensus/scheduler.md (hunk 3 above)
Comment (pro-wh) [reply docs/consensus/scheduler.md:32]: ok

**Pull request merged**